src/models/errors/ims-upload-error.ts

```diff
@@ -0,0 +1,8 @@
+import { ImsError } from './ims-error';
+export class ImsUploadError extends ImsError {
+
+  constructor(err: any) {
+    super('Beim Speichern der Bilder ist ein Fehler aufgetreten.', err);
+    Object.setPrototypeOf(this, ImsUploadError.prototype);
+  }
+}
```

Comment:
Really well done with all the new error classes. This is a really nice and clean concept.